classes/module/Module.php

```diff
@@ -2970,19 +2970,29 @@ public function removeOverride($classname)
                 if (!$override_class->hasProperty($property->getName())) {
                     continue;
                 }
+                $multiline_property_deletion = false;
 
                 // Replace the declaration line by #--remove--#
                 foreach ($override_file as $line_number => &$line_content) {
                     if (preg_match('/(public|private|protected)\s+(static\s+)?(\$)?'.$property->getName().'/i', $line_content)) {
+                        if(!preg_match("/(.*);$/i",trim($line_content))){
```

Comment:
It seems that this does not match with the current coding standards: http://doc.prestashop.com/display/PS16/Coding+Standards

Would you mind changing it to:
`if (!preg_match("/(.*);$/i", trim($line_content))) {`